### ISSUE TITLE
Wrong (lacking) styles are being applied to NavigationBar component

### ISSUE THREAD

**Issue by kheide** [shoutem/ui#209]
This

```
import React, { Component } from 'react'


import {
  View,
  Icon,
  Title,
  Button,
  NavigationBar
} from '@shoutem/ui'


export default class App extends Component {

  render() {
    return (
      <NavigationBar
        leftComponent={<Icon name="sidebar" />}
        centerComponent={<Title>BOOKSHELF</Title>}
        rightComponent={<Button><Icon name="settings" /></Button>}
      />
    )
  }
}
```

Ends up looking like this;

<img width="375" alt="screen shot 2017-03-27 at 10 26 54" src="https://cloud.githubusercontent.com/assets/6345869/24347277/0f81ca58-12d8-11e7-98a0-5587ee7139fe.png">

In direct contradiction to the documentation.

**Comment by kheide** [comment]
Downgrading to 0.10.13 seemed to fix the font-weight / colors, but not the alignments. See screenshot; 

<img width="380" alt="screen shot 2017-03-28 at 19 45 04" src="https://cloud.githubusercontent.com/assets/6345869/24419361/2bc6f0d6-13ef-11e7-812b-e65c13d912d4.png">

**Comment by filiphosko** [comment]
I already submitted a PR for the alignment issue https://github.com/shoutem/ui/pull/211. You can check the explanation in my comment here https://github.com/shoutem/ui/issues/203#issuecomment-290350911.

**Comment by zrumenjak** [comment]
This should be fixed with PR #214, it will be released in the next version.
